### PR TITLE
Add Snyk config

### PR DESCRIPTION
**Pull request by SchSeba**

### CODE REVIEW

**Comment by SchSeba** [comment]
/hold 

need to fix stuff on u/s to support running the unit-tests on prow

**Comment by SchSeba** [comment]
/hold cancel

@wizhaoredhat please take a look

**Comment by wizhaoredhat** [comment]
please fix the WIP commit

File: .golangci.yml [hunk 1]
```diff
@@ -76,10 +76,7 @@ linters:
     - nakedret
     - prealloc
     - revive
-<<<<<<< HEAD
```
Comment (wizhaoredhat) [inline .golangci.yml:79]: Merge conflicts

**Comment by SchSeba** [comment]
Hi @bn222 @wizhaoredhat can you please approve this one now the CI is green

**Pull request merged**